docs/source/notebook-components/weaver_example.ipynb

```diff
@@ -20,29 +20,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {
+    "is_executing": true,
     "pycharm": {
      "name": "#%%\n"
     },
     "tags": []
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Variables:\n",
-      "  WEAVER_TEST_FQDN           pavics.ouranos.ca\n",
-      "  WEAVER_TEST_URL            https://pavics.ouranos.ca/weaver\n",
-      "  WEAVER_TEST_WPS_OUTPUTS    https://pavics.ouranos.ca/wpsoutputs\n",
-      "  WEAVER_TEST_SSL_VERIFY     True\n",
```

Comment:
This output removed because the code stopped creating it?  In general the output should match the code, else it could potentially break Jenkins since pavics-sdi notebooks are run on Jenkins as well.  This notebook is not enabled by default on Jenkins but we should still ensure it works.

Suggestion to test this branch on Jenkins to be sure.  Check the box `TEST_PAVICS_SDI_WEAVER`.  Hit our staging host Boreas that has Weaver enabled.

Reply:
I see. Tests are failing. If the environments are set via docker images, should I simply roll back all changes specific to the notebooks? The point of this PR was to get the notebooks to run on ReadTheDocs.

Reply:
Not sure what you mean about rollback (are you making unnecessary changes not required by ReadTheDocs and you mean to rollback those?) but as long as both Jenkins and ReadTheDocs is not breaking with this change, we are golden.

For Jenkins, simply run it against the current `fix-notebooks` branch of this repo.